meraki/theme.json

```diff
@@ -110,11 +115,26 @@
 	},
 	"styles": {
 		"blocks": {
+			"core/button": {
+				"border": {
+					"radius": "50"
+				},
+				"color": {
+					"background": "var(--wp--preset--color--foreground)",
+					"text": "var(--wp--preset--color--background)"
+				},
+				"typography": {
+					"letterSpacing": "0.05em",
+					"fontSize": "var(--wp--preset--font-size--tiny)",
+					"textTransform": "uppercase",
+					"lineHeight": "1.2"
```

Comment:
I think the lineHeight value chosen introduces an odd padding in the button (top vs bottom)
<img width="190" alt="image" src="https://user-images.githubusercontent.com/146530/163585714-c8cb7dc0-bd16-4846-97fe-4b9038c54104.png">

You might want to tweak that one direction or another to ensure that the padding in the button falls evenly.